File: client-app/src/models/position.ts
export interface Position {
    id: string
    name: string
    createdTime: string,
    employeeIds: string[]
    departmentIds: string[]
  }

export interface PositionCreate {
  name: string
  departmentIds: string[]
}

export interface PositionCreateInDepartment{
  name: string
}

File: EmployeeManagement.Api/Entities/Position.cs
namespace EmployeeManagement.Api;

public class Position
{

    public int Id { get; set; }
    public required string Name { get; set; }
    public DateOnly CreatedTime { get; set; }
    public ICollection<Employee> Employee { get; set; } = [];
    public List<DepartmentPosition>? DepartmentPosition { get; set; }

}
